gapis/api/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -55,8 +55,11 @@
 ¶
   const char kVirtualSwapchainLayerName[] = "VirtualSwapchain";
   const char kGraphicsSpyLayerName[] = "GraphicsSpy";
-  const char kValidationMetaLayerName[] = "VK_LAYER_LUNARG_standard_validation";
   const char* kValidationLayerNames[] = {
+    // Meta layers
+    "VK_LAYER_KHRONOS_validation",
+    "VK_LAYER_LUNARG_standard_validation",
```

Comment:
Do we need both?

Reply:
Yes, because we use these names to search for validation layers loaded by the app, and it may use any validation layer combination.

Reply:
:+1: